client/my-sites/domains/domain-management/domain-overview-pane/style.scss

```diff
@@ -0,0 +1,57 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.domains-overview .item-preview__pane {
+	.item-preview__header .item-preview__header-content .item-preview__header-title-summary {
+		 .item-preview__header-summary-link {
+			color: var(--studio-gray-50, #646970);
+
+			&:hover {
+				text-decoration: none;
+			}
+		}
+
+		.item-preview__header-title {
+			font-family: Recoleta, sans-serif;
+			font-weight: 400;
+			line-height: 1;
+			letter-spacing: normal;
+		}
+	}
+
+	.item-preview__content {
+		padding: 24px;
+		color: var(--studio-gray-100);
+
+		@include break-large {
+			padding: 32px 48px 48px;
+		}
+
+		> * {
+			margin-bottom: auto;
+		}
+	}
+
+	.preview-pane__navigation .section-nav-tabs__list {
+		box-sizing: border-box;
+		padding: 0 8px;
+
+		@include break-large {
+			padding: 0 32px;
+		}
+	}
+
+	.domains-overview__navigation-header.navigation-header {
+		padding: 0;
+		margin-bottom: 24px;
+		border: 0;
+
+		@include break-large {
+			margin-bottom: 32px;
+		}
+
+		.navigation-header__main {
+			padding-inline: 0;
+		}
+	}
+}
```

Comment:
I've noticed that in the current domain-overview screen, the accordions go full width in mobile view. But here they don't. So the starting of the right-column cards look a bit misaligned -

Existing -

<img width="432" alt="Screenshot 2024-12-10 at 4 47 25 PM" src="https://github.com/user-attachments/assets/81f9aaa1-f5eb-4f66-8aae-a898bc9ac80c">

New -

<img width="432" alt="Screenshot 2024-12-10 at 4 48 48 PM" src="https://github.com/user-attachments/assets/3d840e37-e62f-4741-9727-1b3df1fefae1">

Reply:
Switching to full-width on mobile managed to solve these issues. Could you please give it another look?

![image](https://github.com/user-attachments/assets/1b611516-995c-4044-8980-1e2ace160ed2)